resources.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: resources
+title: Resources
+permalink: /resources/
+weight: 20
+sections:
+  - resources:
+    - name: GitHub
+      link: https://github.com/indico/indico
+      logo: github.png
+      summary: Visit our GitHub project site. You can report issues there.
+    - name: Community forum
+      link: https://talk.getindico.io
+      logo: discourse.png
+      summary: Join other Indico administrators on our forum to seek advice or discuss your ideas
+    - name: Getting started
+      link: https://docs.getindico.io
+      logo: indico.png
+      summary: Learn how to set up an Indico instance or discover how to use the Indico API
+    - name: User's guide
+      link: https://learn.getindico.io
+      logo: indico.png
+      summary: Discover all Indico features with video walkthroughs and tutorials
+    - name: Contact
+      link: https://getindico.io/contact
+      logo: indico.png
+      summary: Get in touch with indico developers and maintainers of CERN instance
```

Comment:
Indico with uppercase-i; and not sure if mentioning the CERN instance here is useful (but if you want to keep it: +the (2x))